doc/tutorial/clean_mix.py

```diff
@@ -1,7 +1,7 @@
 from doit.task import clean_targets
 
 def simple():
-    print "ok"
```

Comment:
python-modernize is quite handy for this, for example for prints you can use this:
```
python-modernize -w -f libmodernize.fixes.fix_print dodo.py
```
http://python-modernize.readthedocs.org/en/latest/